client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/ExposuresAndFlights.tsx

```diff
@@ -44,6 +50,12 @@ const ExposuresAndFlights : React.FC<Props> = ({ id, onSubmit }: Props): JSX.Ele
   const doesHaveConfirmedExposures = (checkedExposures: Exposure[]) => checkedExposures.some(exposure => exposure.wasConfirmedExposure)
   const doesHaveFlights = (checkedExposures: Exposure[]) => checkedExposures.some(exposure => exposure.wasAbroad)
 
+  React.useEffect(() => {
```

Comment:
is it necessary? for now, it always true so trigger does nothing

Reply:
As of this pr exposures and flights tab doesn't have validation schema, when one will be implemented this will have meaning, just thinking of the next guy that's all